vividus-to-zephyr-exporter/src/main/java/org/vividus/zephyr/VividusToZephyrExporterApplication.java

```diff
@@ -17,24 +17,30 @@
 package org.vividus.zephyr;
 
 import java.io.IOException;
-import java.util.Properties;
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.boot.context.properties.EnableConfigurationProperties;
 import org.springframework.context.ApplicationContext;
-import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Import;
 import org.springframework.context.annotation.ImportResource;
+import org.springframework.context.annotation.PropertySource;
+import org.vividus.common.VividusExporterPropertyParser;
 import org.vividus.jira.JiraConfigurationException;
-import org.vividus.util.property.PropertyParser;
 import org.vividus.zephyr.configuration.ZephyrExporterConfiguration;
 import org.vividus.zephyr.configuration.ZephyrExporterProperties;
 import org.vividus.zephyr.exporter.ZephyrExporter;
 
 @SpringBootApplication
-@ImportResource(locations = { "org/vividus/zephyr/spring.xml", "org/vividus/http/client/spring.xml",
-        "org/vividus/jira/spring.xml" })
-@EnableConfigurationProperties({ ZephyrExporterConfiguration.class, ZephyrExporterProperties.class })
+@ImportResource(locations = { "org/vividus/jira/spring.xml", "org/vividus/http/client/spring.xml",
+        "org/vividus/zephyr/spring.xml", "org/vividus/util/spring.xml" })
```

Comment:
just append the missing spring config as a last one, do not introduce the noisy changes